cli/manage.go

```diff
@@ -125,7 +125,7 @@ func manage(c *cli.Context) {
 		log.Fatal("--heartbeat should be an unsigned integer and greater than 0")
 	}
 	options := &cluster.Options{
-		TLSConfig:       tlsConfig,
```

Comment:
Will revert to its initial name, this a the result of tinkering with TLS in discovery.